### PR TITLE
Update the README to clarify component ownership

### PR DESCRIPTION
**Pull request by LorenzMeier**
This represents more closely the actual maintenance model. To be discussed on the next dev call.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,36 +45,24 @@ The PX4 Dev Team syncs up on a [weekly dev call](https://dev.px4.io/en/contribut
...
   * [Multicopter Flight Control](https://github.com/PX4/Firmware/labels/multicopter)
-    * [Dennis Mannhart](https://github.com/Stifael)
+    * [Mathieu Bresciani](https://github.com/bresch)
+  * [Multicopter Software Architecture](https://github.com/PX4/Firmware/labels/multicopter)
     * [Matthias Grob](https://github.com/MaEtUgR)
   * [VTOL Flight Control](https://github.com/PX4/Firmware/labels/vtol)
-    * [Daniel Agar](https://github.com/dagar)
-    * [Mathieu Bresciani](https://github.com/bresch)
-    * [Sander Smeets](https://github.com/sanderux)
     * [Roman Bapst](https://github.com/tumbili)
```
Comment (dagar) [inline README.md]:
```suggestion
    * [Roman Bapst](https://github.com/RomanBapst)
```

File: README.md [hunk 2]
```diff
@@ -45,36 +45,24 @@ The PX4 Dev Team syncs up on a [weekly dev call](https://dev.px4.io/en/contribut
...
-    * [Andreas Antener](https://github.com/AndreasAntener)
   * [Fixed Wing Flight Control](https://github.com/PX4/Firmware/labels/fixedwing)
-    * [Daniel Agar](https://github.com/dagar)
-    * [Paul Riseborough](https://github.com/priseborough)
-  * Racers - [Matthias Grob](https://github.com/MaEtUgR)
-  * OS / drivers - [David Sidrane](https://github.com/davids5)
-  * [UAVCAN](https://github.com/PX4/Firmware/labels/uavcan) / Industrial - [Pavel Kirienko](https://github.com/pavel-kirienko)
-  * [State Estimation](https://github.com/PX4/Firmware/issues?q=is%3Aopen+is%3Aissue+label%3A%22state+estimation%22) - [James Goppert](https://github.com/jgoppert), [Paul Riseborough](https://github.com/priseborough)
+    * [Roman Bapst](https://github.com/tumbili)
```
Comment (dagar) [inline README.md]:
```suggestion
    * [Roman Bapst](https://github.com/RomanBapst)
```

File: README.md [hunk 3]
```diff
@@ -45,36 +45,24 @@ The PX4 Dev Team syncs up on a [weekly dev call](https://dev.px4.io/en/contribut
...
+  * OS / NuttX [David Sidrane](https://github.com/davids5)
+  * Driver Architecture [Daniel Agar](https://github.com/dagar)
+  * Commander Architecture [Julian Oes](https://github.com/julianoes)
```
Comment (julianoes) [inline README.md]: This can hopefully be true in the future but for now I don't own any of it yet.

Comment (LorenzMeier) [reply README.md]: It means you have the responsibility now and it is your job to build a roadmap and execute on it.

**Pull request merged**